src/vite.config.js

```diff
@@ -0,0 +1,37 @@
+import { defineConfig, loadEnv,  } from 'vite'
+import vue from '@vitejs/plugin-vue'
+import laravel from 'laravel-vite-plugin'
+import { networkInterfaces } from 'os'
+
+export default ({ mode }) => {
+    process.env = { ...process.env, ...loadEnv(mode, process.cwd()) }
+
+    return defineConfig({
+        build: {
+            outDir: './public/build/',
+        },
+        server: {
+            host: Object.values(networkInterfaces()).flat().find(i => i.family === 'IPv4' && !i.internal).address,
+            port: process.env.VITE_PORT,
```

Comment:
I think port is not needed. Default will be used.

Reply:
A port is needed, without a port it does not work.

Reply:
I don't believe. What error do you have? Other projects run fine without port, e.g. Toby.

Reply:
Anyway, let's keep it as it is. It will be more configurable then.